ui/src/sources/components/InfluxTableRow.tsx

```diff
@@ -27,6 +27,14 @@ interface Props {
 }
 
 class InfluxTableRow extends PureComponent<Props & WithRouterProps> {
+  private connectionLink: React.RefObject<ConnectionLink> = React.createRef()
+
+  public componentDidUpdate() {
+    if (this.connectionLink.current) {
```

Comment:
could maybe store this.connectionLink.current in a variable

Reply:
if we use it again then I would agree, right now seems like only increases the signal to noise ratio.